patches/tModLoader/Terraria/Main.cs.patch

```diff
@@ -1029,12 +1029,10 @@
  					CustomCurrencyManager.GetPriceText(hoverItem.shopSpecialCurrency, array, ref numLines, num5);
  					color = new Microsoft.Xna.Framework.Color((byte)(255f * num4), (byte)(255f * num4), (byte)(255f * num4), a);
  				}
-@@ -14912,7 +_,8 @@
- 						array[numLines] = Lang.tip[49].Value + " " + text;
+@@ -14913,6 +_,7 @@
  					else
  						array[numLines] = Lang.tip[50].Value + " " + text;
--
-+						
```

Comment:
I still see some redundant patch changes here.
Just in general we try to make sure to only edit the necessary stuff and keeping patches small is a necessity